### DISCUSSION TITLE
Newly added row initial display state

### DISCUSSION THREAD

**Discussion by ollie-campbell** [grafana/grafana#62178] [Dashboards, Panels & Visualizations]
When you add a new row to a dashboard it gets added in an un-rolled state. In this state it would automatically consume any panels at the top of a dashboard that are not included in their own rows.

I have panels at the top of many dashboards that I want to stay at the top of a dashboard outside of a row so that they can't be hidden by accident.

Can Grafana rows be changed so that they are added to dashboards in a rolled-up state by default? This way, panels at the top of a dashboard aren't automatically consumed by this new row?

It feels like a quick and easy change that wouldn't be detrimental to existing users....

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82921. Feel free to continue the discussion around this there. Thank you!
